api/node.go

```diff
@@ -191,14 +184,6 @@ func removeNodeHandler(w http.ResponseWriter, r *http.Request, t auth.Token) (er
 		return permission.ErrUnauthorized
 	}
 	removeIaaS, _ := strconv.ParseBool(r.URL.Query().Get("remove-iaas"))
```

Comment:
Just a minor thing about readability... you're using `removeIaaS`  ~20 lines later